src/app/modules/gb-export-module/gb-export.component.html

```diff
@@ -7,46 +7,80 @@ <h5>Create new export for the currently selected data</h5>
     </div>
 
     <div class="gb-export-format-container">
+      <form novalidate #exportJobForm="ngForm">
 
-      <!-- area to indicate which data formats to export -->
-      <div class="gb-export-format-header">Select data types and formats to export:</div>
-      <div *ngFor="let dataType of exportDataTypes">
-      <span class="row">
-        <span class="ui-grid ui-grid-responsive ui-fluid gb-export-format-item">
-          <p-checkbox [(ngModel)]="dataType.checked" binary="true"></p-checkbox>
-          {{dataType.name}}
+      <div class="container-fluid">
+        <!-- area to indicate which data formats to export -->
+        <div class="gb-export-format-header row">Select data types and formats to export:</div>
+        <span *ngFor="let dataType of exportDataTypes" class="row">
+          <span class="ui-grid ui-grid-responsive ui-fluid gb-export-format-item">
+            <p-checkbox [(ngModel)]="dataType.checked" name="dataTypes" binary="true"></p-checkbox>
+            {{dataType.name}}
+          </span>
+          (&nbsp;
+          <span *ngFor="let fileFormat of dataType.fileFormats">
+            <p-checkbox [(ngModel)]="fileFormat.checked" name="fileFormats" binary="true"></p-checkbox>
+            {{fileFormat.name}} &nbsp;
+          </span>
+          )
         </span>
-        (&nbsp;
-        <span *ngFor="let fileFormat of dataType.fileFormats">
-          <p-checkbox [(ngModel)]="fileFormat.checked" binary="true"></p-checkbox>
-          {{fileFormat.name}} &nbsp;
-        </span>
-        )
-      </span>
+        <div>
+          <small class="form-text text-danger" *ngIf="!isFileFormatSelected">
+            No data type or file format selected.
+          </small>
+        </div>
       </div>
 
       <!-- export name input and button -->
       <div class="form-inline gb-export-btn-container">
-        <input id="exportJobNameInput" type="text" class="form-text"
-               placeholder="Export job name"
-               (drop)="onExportJobNameInputDrop($event)"
-               [(ngModel)]="exportJobName">
-        <span>&nbsp;</span>
-        <button type="button" class="btn btn-primary btn-sm"
-                (click)="createExportJob()">Create Export</button>
-        <span>&nbsp;</span>
+        <div>
+          <input id="exportJobNameInput" type="text" class="form-control"
+                 name="exportJobNameInput"
+                 #exportJobNameInput="ngModel"
+                 placeholder="Export job name"
+                 (drop)="onExportJobNameInputDrop($event)"
+                 required
+                 minlength="1"
+                 maxlength="200"
+                 gbNonEmpty
+                 [gbExcludeValues]="jobNames"
+                 [(ngModel)]="exportJobName">
```

Comment:
Maybe it would be useful to validate also the special characters? Currently when I use characters like `+` or `#` etc. they are just replaced by white spaces. When I use only them - the whole name is replaced with job id and event though the job is created I the error see message: "A job name is required. "
This is unlikely that the user will provide such a name, but possible.

Reply:
Interesting, do you know where that happens?

Reply:
Looks like `#` (and every character after #) is removed automatically from the request query string, but others not sure

Reply:
but maybe it's better to merge this PR and report special characters validation as a separate issue?

Reply:
It should be fixed now by using [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) on the job name.